packages/expo-router/src/native-tabs/types.ts

```diff
@@ -153,6 +153,10 @@ export interface NativeTabsProps extends PropsWithChildren {
    * The background color of every badge in the tab bar.
    */
   badgeBackgroundColor?: ColorValue;
+  /**
+   * When set to `true`, hides the tab bar.
```

Comment:
```suggestion
   * When set to `true`, hides the tab bar.
   * 
   * @default false
```

Let's add a default value for clarity.